Engine/Tests/StepTests.cs

```diff
@@ -41,8 +41,8 @@ public void PassingStepShouldBeSuccessful()
 
             Assert.IsType<SuccessStepResult>(result);
             Assert.Same(step, result.Step);
-            Assert.True(step.Instance.Cast<TestKoan>().WasSetup, "Koan was unexpectantly setup");
-            Assert.True(step.Instance.Cast<TestKoan>().WasToreDown, "Koan was unexpectantly tore down");
+            Assert.True(step.Instance.Cast<TestKoan>().WasSetup, "Koan was unexpectedly setup");
+            Assert.True(step.Instance.Cast<TestKoan>().WasToreDown, "Koan was unexpectedly tore down");
```

Comment:
Heyy, not a maintainer but I think [unexpectantly](https://en.wiktionary.org/wiki/unexpectantly) is correct.

Reply:
While `unexpectantly` seems to be used sometimes, the wiktionary site states that its usage is "proscribed" (i.e. it should not be used). I found multiple other sources backing this up, so `unexpectedly` seems to be the better choice here.

Reply:
ohh I see. I thought it was unexpectedly that was labelled as proscribed.
Makes sense thinking about it now.